### PR TITLE
First version of the styling for manifest and APIs in versions

### PR DESCRIPTION
**Pull request by annloouise**

### CODE REVIEW

File: src/views/apis-in-version/apis-in-version.js [hunk 1]
```diff
@@ -1,11 +1,71 @@
...
+              How to use
+            </div>
+            <div class="description-text">
+              Select an API and a version to display the functions included in that specific version for the selected API.
+            </div>
+          </div>
+        </div>
+        <div class="bottom-content">
+          ${until(/* eslint-disable indent */
+            getChangesModel().then(model => model.getLayout().then((layout) => {
```
Comment (bjornwiberg) [inline src/views/apis-in-version/apis-in-version.js]:
```suggestion
            getChangesModel().then(model => model.getLayout().then(({qHyperCube: { qPivotDataPages: [data]}}) => {
              if (data) {
```

File: src/views/apis-in-version/apis-in-version.js [hunk 2]
```diff
@@ -1,11 +1,71 @@
...
+                const data = layout.qHyperCube.qPivotDataPages[0]
```
Comment (bjornwiberg) [inline src/views/apis-in-version/apis-in-version.js]:
```suggestion
```

File: src/views/manifest/manifest.js [hunk 3]
```diff
@@ -1,11 +1,71 @@
...
+              How to use
+            </div>
+            <div class="description-text">
+              Select a version to see the list of included components and APIs, or select a component to see its versioning history between releases.
+            </div>
+          </div>
+        </div>
+        <div class="bottom-content">
+          ${until(/* eslint-disable indent */
+            getChangesModel().then(model => model.getLayout().then((layout) => {
```
Comment (bjornwiberg) [inline src/views/manifest/manifest.js]:
```suggestion
            getChangesModel().then(model => model.getLayout().then(({qHyperCube: { qPivotDataPages: [data]}}) => {
              if (data) {
```

File: src/views/manifest/manifest.js [hunk 4]
```diff
@@ -1,11 +1,71 @@
...
+                const data = layout.qHyperCube.qPivotDataPages[0]
```
Comment (bjornwiberg) [inline src/views/manifest/manifest.js]:
```suggestion
```

File: src/views/manifest/manifest.js [hunk 5]
```diff
@@ -1,11 +1,71 @@
...
+                  ? layout.qHyperCube.qPivotDataPages[0] : null;
```
Comment (bjornwiberg) [inline src/views/manifest/manifest.js]:
```suggestion
```

File: src/views/apis-in-version/apis-in-version.js [hunk 6]
```diff
@@ -1,11 +1,71 @@
...
+                          ${repeat(rowHeaders, null, (header, index) => html`<tr>
+                              <th>${header}</th>
+                              ${repeat(data.qData[index], null, item => html`<td>${item.qText}</td>`)}
+                            </tr>`)}
+                        </tbody>
+                      </table>
+                    </div>
+                  `;
+                }
+                return html`NO DATA`;
```
Comment (bjornwiberg) [inline src/views/apis-in-version/apis-in-version.js]:
```suggestion

                return html`NO DATA`;
```

File: src/views/manifest/manifest.js [hunk 7]
```diff
@@ -1,11 +1,71 @@
...
+                          ${repeat(rowHeaders, null, (header, index) => html`<tr>
+                              <th>${header}</th>
+                              ${repeat(data.qData[index], null, item => html`<td>${item.qText}</td>`)}
+                            </tr>`)}
+                        </tbody>
+                      </table>
+                    </div>
+                  `;
+                }
+                return html`NO DATA`;
```
Comment (bjornwiberg) [inline src/views/manifest/manifest.js]:
```suggestion

                return html`NO DATA`;
```

File: src/views/manifest/manifest.js [hunk 8]
```diff
@@ -1,11 +1,71 @@
...
+                  const tblHeaders = new Set(data.qTop.map(el => (el.qText || '')));
+                  const rowHeaders = new Set(data.qLeft.map(el => (el.qText || '')));
+
+                  return html`
+                    <div class="table-wrapper">
+                      <table class="ai-table sticky">
+                        <thead>
+                          <tr>
+                          <th></th>
+                          ${repeat(tblHeaders, null, header => html`<th>${header}</th>`)}
```
Comment (bjornwiberg) [inline src/views/manifest/manifest.js]: add scope for accessibility

```suggestion
                          ${repeat(tblHeaders, null, header => html`<th scope="col">${header}</th>`)}
```

File: src/views/manifest/manifest.js [hunk 9]
```diff
@@ -1,11 +1,71 @@
...
+                          </tr>
+                        </thead>
+                        <tbody>
+                          ${repeat(rowHeaders, null, (header, index) => html`<tr>
+                              <th>${header}</th>
```
Comment (bjornwiberg) [inline src/views/manifest/manifest.js]: add scope for accessibility

```suggestion
                              <th scope="row">${header}</th>
```

File: src/views/apis-in-version/apis-in-version.js [hunk 10]
```diff
@@ -1,11 +1,71 @@
...
+                  const tblHeaders = new Set(data.qTop.map(el => (el.qText || '')));
+                  const rowHeaders = new Set(data.qLeft.map(el => (el.qText || '')));
+
+                  return html`
+                    <div class="table-wrapper">
+                      <table class="ai-table sticky">
+                        <thead>
+                          <tr>
+                          <th></th>
+                          ${repeat(tblHeaders, null, header => html`<th>${header}</th>`)}
```
Comment (bjornwiberg) [inline src/views/apis-in-version/apis-in-version.js]: add scope for accessibility

```suggestion
                          ${repeat(tblHeaders, null, header => html`<th scope="col">${header}</th>`)}
```

File: src/views/apis-in-version/apis-in-version.js [hunk 11]
```diff
@@ -1,11 +1,71 @@
...
+                          </tr>
+                        </thead>
+                        <tbody>
+                          ${repeat(rowHeaders, null, (header, index) => html`<tr>
+                              <th>${header}</th>
```
Comment (bjornwiberg) [inline src/views/apis-in-version/apis-in-version.js]: add scope for accessibility

```suggestion
                              <th scope="row">${header}</th>
```

File: src/views/apis-in-version/apis-in-version.js [hunk 12]
```diff
@@ -1,11 +1,71 @@
...
+                              ${repeat(data.qData[index], null, item => html`<td>${item.qText}</td>`)}
+                            </tr>`)}
+                        </tbody>
+                      </table>
+                    </div>
+                  `;
+                }
+                return html`NO DATA`;
+              })), html`Loading Data...`,
```
Comment (bjornwiberg) [inline src/views/apis-in-version/apis-in-version.js]:
```suggestion
              })), 'Loading Data...',
```

File: src/views/manifest/manifest.js [hunk 13]
```diff
@@ -1,11 +1,71 @@
...
+                              ${repeat(data.qData[index], null, item => html`<td>${item.qText}</td>`)}
+                            </tr>`)}
+                        </tbody>
+                      </table>
+                    </div>
+                  `;
+                }
+                return html`NO DATA`;
+              })), html`Loading Data...`,
```
Comment (bjornwiberg) [inline src/views/manifest/manifest.js]:
```suggestion
              })), 'Loading Data...',
```

File: src/views/apis-in-version/apis-in-version.js (hunk 1 above)
Comment (annloouise) [reply src/views/apis-in-version/apis-in-version.js]: This is just to include some "dummy" data, that can be changed in anyway later on

File: src/views/manifest/manifest.js (hunk 13 above)
Comment (annloouise) [reply src/views/manifest/manifest.js]: Fixed in my next PR

File: src/views/manifest/manifest.js (hunk 7 above)
Comment (annloouise) [reply src/views/manifest/manifest.js]: Fixed in my next PR

File: src/views/apis-in-version/apis-in-version.js (hunk 12 above)
Comment (annloouise) [reply src/views/apis-in-version/apis-in-version.js]: Fixed in my next PR

File: src/views/apis-in-version/apis-in-version.js (hunk 6 above)
Comment (annloouise) [reply src/views/apis-in-version/apis-in-version.js]: Fixed in my next PR

File: src/views/manifest/manifest.js (hunk 9 above)
Comment (annloouise) [reply src/views/manifest/manifest.js]: Fixed in my next PR

File: src/views/manifest/manifest.js (hunk 8 above)
Comment (annloouise) [reply src/views/manifest/manifest.js]: Fixed in my next PR

File: src/views/manifest/manifest.js (hunk 5 above)
Comment (annloouise) [reply src/views/manifest/manifest.js]: Won't fix, can be changed when we load the data for real

File: src/views/manifest/manifest.js (hunk 4 above)
Comment (annloouise) [reply src/views/manifest/manifest.js]: Won't fix, can be changed when we load the data for real

File: src/views/manifest/manifest.js (hunk 3 above)
Comment (annloouise) [reply src/views/manifest/manifest.js]: Won't fix, can be changed when we load the data for real

File: src/views/apis-in-version/apis-in-version.js (hunk 11 above)
Comment (annloouise) [reply src/views/apis-in-version/apis-in-version.js]: Fixed in my next PR

File: src/views/apis-in-version/apis-in-version.js (hunk 10 above)
Comment (annloouise) [reply src/views/apis-in-version/apis-in-version.js]: Fixed in my next PR

File: src/views/apis-in-version/apis-in-version.js (hunk 2 above)
Comment (annloouise) [reply src/views/apis-in-version/apis-in-version.js]: Won't fix, can be changed when we load the data for real

File: src/views/apis-in-version/apis-in-version.js (hunk 6 above)
Comment (aihazm) [reply src/views/apis-in-version/apis-in-version.js]: your suggestion (removing) will probably not work

**Pull request merged**